tests/unit_tests/test_data_structure.py

```diff
@@ -9,7 +9,11 @@
 from openscap_report.scap_results_parser.data_structures import Remediation
 from tests.unit_tests.test_scap_result_parser import get_parser
 
-from ..constants import PATH_TO_ARF
+from ..constants import (PATH_TO_ARF, PATH_TO_ARF_SCANNED_ON_CONTAINER,
+                         PATH_TO_SIMPLE_RULE_FAIL_ARF,
+                         PATH_TO_SIMPLE_RULE_FAIL_XCCDF,
+                         PATH_TO_SIMPLE_RULE_PASS_ARF,
+                         PATH_TO_SIMPLE_RULE_PASS_XCCDF, PATH_TO_XCCDF)
```

Comment:
I think that having each item on a new line would be a better code style

Reply:
I am using to format and sort of imports tool name [isort](https://pycqa.github.io/isort/index.html). This is the output of `isort`. Isort is used in the test suite to avoid unused imports or not formatted imports.  I think formatting with `isort` is good.

Reply:
aha, clever! thanks for explanation

If it's a generated code then it doesn't make sense to change it